tests/functional/setups/manu_sneddon_2d.py

```diff
@@ -165,77 +171,71 @@ def get_bc_val(
 
 
 def assign_bem(
-    g: pp.Grid,
+    grid: pp.Grid,
     h: float,
     bound_faces: np.ndarray,
-    theta: float,
+    alpha: float,
     bem_centers: np.ndarray,
     u_a: np.ndarray,
     poi: float,
 ) -> np.ndarray:
-    """
-    Compute analytical displacement using the BEM method for the Sneddon problem.
+    """Computes semi-analytical displacement values using the BEM for the Sneddon
+    problem.
 
     Parameter
-        g: Subdomain grid
-        h: bem segment length
-        bound_faces: boundary faces
-        theta: fracture orientation
-        bem_centers: bem segments centers
-        u_a: Sneddon's analytical relative normal displacement
-        poi: Poisson ratio
+        grid: The matrix grid.
+        h: BEM segment length.
+        bound_faces: Array of indices of boundary faces of ``grid``.
+        alpha: Fracture orientation.
+        bem_centers: BEM segments centers.
+        u_a: Sneddon's analytical relative normal displacement.
+        poi: Poisson ratio.
 
     Return:
-        Semi-analytical boundary displacement values
+        Semi-analytical boundary displacement values.
+
     """
 
-    bc_val = np.zeros((g.dim, g.num_faces))
+    bc_val = np.zeros((grid.dim, grid.num_faces))
 
-    alpha = np.pi / 2 - theta
+    alpha = np.pi / 2 - alpha
 
-    bound_face_centers = g.face_centers[:, bound_faces]
+    bound_face_centers = grid.face_centers[:, bound_faces]
 
     for i in range(0, u_a.size):
-
         new_bound_face_centers = transform(bem_centers[:, i], bound_face_centers, alpha)
 
         u_bound = get_bc_val(
-            g, bound_faces, new_bound_face_centers, h, poi, alpha, u_a[i]
+            grid, bound_faces, new_bound_face_centers, h, poi, alpha, u_a[i]
         )
 
         bc_val += u_bound
 
     return bc_val
 
 
-class ManuExactSneddon2dSetup:
-    """
-    Class for setting up the analytical solution for the pressurized fracture problem.
-    
-    
-    """
-  
-    def __init__(self, setup):
-        # Initialize private variables from the setup dictionary
-        self.p0 = setup.get("p0")
-        self.theta = setup.get("theta")
-
-        self.a = setup.get("a")
-        self.shear_modulus  = setup.get("material_constants").get("solid").shear_modulus
-        self.poi = setup.get("poi")
-        self.length = setup.get("length")
-        self.height = setup.get("height")
-         
-  
-    def exact_sol_global(self, sd):
-        """
-        Compute the analytical solution for the pressurized fracture problem in question.
-        
+class ManuSneddonExactSolution2d:
```

Comment:
Do we really want "Manu", since its an analytical, rather than manufactured, solution?